f/fluentd/build_info.json

```diff
@@ -13,6 +13,6 @@
     "dir": "1.14.5_es7.17.2_ubi8"
   },
   "v*.*.*": {
-    "dir": "1.14.5_es7.17.2_ubi8"
+    "build_script": "fluentd_ubi_9.3.sh"
```

Comment:
so we are keeping docker support here? @pooja0805

Reply:
I am currently working on Dockerfile for latest version